src/input/mpegts/satip/satip.c

```diff
@@ -203,6 +203,20 @@ const idclass_t satip_device_class =
       .off      = offsetof(satip_device_t, sd_pilot_on),
     },
     {
+      .type     = PT_BOOL,
+      .id       = "fritzquirks",
+      .name     = "Enable FRITZ!Box-workarounds",
```

Comment:
I don't like it so much. What about to rename this as 'pids21' which is more universal.